charts/back8sup/values.yaml

```
@@ -12,7 +12,9 @@ image:
   # image.pullPolicy -- set the image pullPolicy
   pullPolicy: IfNotPresent
   # image.tag -- set the tag of the image
-  tag: "v0.1.3"
+  # Specify a tag to override which version of timed to deploy.
+  # If no tag is specified the appVersion from Chart.yaml is used as tag.
+  # tag:
```

```suggestion
  # Specify a tag to override which version of timed to deploy.
  # If no tag is specified the appVersion from Chart.yaml is used as tag.
  tag: ""
```

The caasperli chart example does it like this:
```
  # image.tag -- Overrides the image tag whose default is the chart version.
  tag: ""
```

---

Well cool. But not the `charts/timed` took it from there.